x-pack/legacy/plugins/reporting/export_types/csv/server/lib/generate_csv.ts

```diff
@@ -82,11 +86,20 @@ export function createGenerateCsv(logger: Logger) {
     const size = builder.getSizeInBytes();
     logger.debug(`finished generating, total size in bytes: ${size}`);
 
+    if (csvContainsFormulas && settings.escapeFormulaValues) {
+      warnings.push(
+        i18n.translate('xpack.reporting.exportTypes.csv.generateCsv.escapedFormulaValues', {
+          defaultMessage: 'CSV may contain formulas whose values have been escaped',
```

Comment:
This doesn't feel like an "Error" condition to me, but moreso an informational message

![image](https://user-images.githubusercontent.com/3493255/79569521-bbeb7f00-8085-11ea-9676-d503de43eba2.png)

Reply:
Yeah, we can probably just remove that second line since we have more information in the Job details. We have a task to come back and consolidate all of these warning messages since we now have a method of sending generic messages.

Reply:
We'll follow up with this in another PR